options/options_settable_test.cc

```diff
@@ -34,6 +34,8 @@ namespace ROCKSDB_NAMESPACE {
 #if defined OS_LINUX || defined OS_WIN
 #ifndef __clang__
 #ifndef ROCKSDB_UBSAN_RUN
+// status check adds CXX flag -fno-elide-constructors which fails this test.
+#ifndef ROCKSDB_ASSERT_STATUS_CHECKED
```

Comment:
Tests in this file fail when ROCKSDB_ASSERT_STATUS_CHECKED is enabled, so I disabled these tests for now.

Reply:
For some reason, I'm only seeing one test failure. Are you seeing more failures for disabling the whole test?

```
[ RUN      ] OptionsSettableTest.ColumnFamilyOptionsAllFieldsSettable
options/options_settable_test.cc:568: Failure
Expected equality of these values:
  unset_bytes_base
    Which is: 115
  NumUnsetBytes(new_options_ptr, sizeof(ColumnFamilyOptions), kColumnFamilyOptionsExcluded)
```

Reply:
You are right, only one test failed. Updated to only skip that test.